index.js

```diff
@@ -19,8 +19,23 @@ module.exports = function (session) {
 
     this.conString = options.conString || process.env.DATABASE_URL;
```

Comment:
Remove  the `this.conString` property, it's no longer needed if support for older pg-modules are dropped.